apps/benchmark/src/index.ts

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { type PipelineMeasure } from '@jvalue/jayvee-execution';
+
+import benchmarkDefinitions from './benchmark_definitions.json';
+import { createInterpreter, runBenchmark } from './run';
+
+function measureCompare(
+  actual: PipelineMeasure,
+  expected: PipelineMeasure,
+  allowedDeviationFactor: number,
+): '=' | '<' | '>' {
```

Comment:
I'd use `tooSlow` instead of `<` and `tooFast` instead of `>`. Makes it a bit easier to read is the calling code.